BUILD.md

```diff
@@ -535,6 +535,25 @@ Run tests and generate report:
 ``` shell
 meson setup -Db_coverage=true build
 meson test -C build
-cd build
-ninja coverage-html
+ninja -C build coverage-html
+```
+
+### Static analysis report
+
+Prerequisites:
+
```

Comment:
Suggest adding `python3-setuptools` to the install packages, followed by a generic-OS Python section: `sudo pip3 install beautifulsoup4 html5lib`

(unless all of that's unnecessary on the command-line?)

Reply:
It's unnecessary :) These packages are needed to count the number of issues in HTML report - which is useful only on CI.

I think we had these packages installed via pip because they were missing in Ubuntu 18.04, but now we have them in normal repo, so there's no reason to invoke pip at all. As a bonus, Ubuntu package has correct dependencies, so manual installation of `html5lib` is no longer needed :)